README.md

```diff
@@ -57,7 +57,7 @@ The cached wms tiles will be written to ``./configuration/cache_data`` externall
 defined by the mapproxy.yaml.
 
 **Note** that the mapproxy containerised application will run as the user that
-owns the /mapproxy folder.
+owns the /mapproxy folder. The UID:GID of the process will be 1000:10001. If you serve existing ``./configuration`` folder, you need to set the folder permission with `chown -R 1000:10001 ./configuration` from this directory.
```

Comment:
We already have container security context in place, so we need to tell the user if they supply their own volume and have different owner mapping.